api/v3/Contact.php

```diff
@@ -758,10 +758,18 @@ function civicrm_api3_contact_getquick($params) {
     )
   );
 
+  $table_names = [
```

Comment:
why don't we just use names rather than these dumb abbreviations all the way!

Reply:
Agree, but the patch was getting big!

Reply:
ok

Reply:
Remember that this is a deprecated api, so there's only so much work I want to put into it.